product-development/README.md

```diff
@@ -104,57 +117,99 @@ Example:
 - Refer to [Analytics](https://github.com/tilt-dev/company-private/tree/master/analytics) to determine how to collect metrics. Contact Victor if you're unsure how to use the analytics software or collect the data that you're interested in.
 ```
 
-## Picking an epic to work on
-When a Tilt engineer is free, they should [pick a high priority unstarted epic](#picking-a-high-priority-unstarted-epic) to work on, or join another engineer (or engineers) already working on an `In Progress` epic. (They might consider [fixing a bug instead](../development/README.md#assign-yourself-a-bug).) Engineers should self-organize and coordinate timelines ad hoc when picking epics. There are no stable teams, no timeboxed sprints, and no synced sprints across teams. Engineers should consider:
-- Working in at least pairs on a given epic, to better facilitate product and code collaboration, avoiding working in siloes.
-- Having a dedicated review Tilter per epic, who may be responsible for code review, content review and/or general deliverable review, who may not necessarily be collaborating during the design and implementation part of the epic, but is responsible for reviewing work.
-- Not working on too many epics concurrently, in order to focus on a smaller number of tasks to reduce context-switching.
+## How an Epic Starts
 
-When you have picked an epic to work on, move it into the `In Progress` state. Remove any existing owner. Make yourself the owner. You are now the [DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4) for the epic, and are responsible for driving the epic to completion. An engineer should be the DRI of at most one `In Progress` epic at any time. An epic should only have one owner at any time.
+### DRIs
 
-Dan and Nick will typically not be DRIs on epics, in order to encourage all Tilters to have more responsibility for innovation and execution of day to day work.
+Epics are assigned to a
+[DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4). The
+DRI is responsible for driving the epic to completion. An engineer should be the
+DRI of at most one `In Progress` epic at any time. An epic should only have one
+owner at any time.
```

Comment:
```suggestion
Epics are assigned to a
[DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4). The
DRI is responsible for driving the epic to completion.
```

Simplifying this section. The removed sentences are legacy and more clearly explained in the new subsequent sections.

Reply:
simplified even more!